injector.js

```diff
@@ -2,14 +2,14 @@
 
 var _ = require('lodash');
 
-function Injector(options) {
+function Injector (options) {
   Injector._ensureRequireJS();
 
-  this.options = _.extend({ context: Injector.DEFAULT_CONTEXT }, options || {});
+  this.options = _.assignIn({ context: Injector.DEFAULT_CONTEXT }, options || {});
```

Comment:
I would use `assign` everywhere to be more compliant with `Object.assign` :)

```suggestion
  this.options = _.assign({ context: Injector.DEFAULT_CONTEXT }, options || {});
```